### DISCUSSION TITLE
【第3回】2024年03月24日

### DISCUSSION THREAD

**Discussion by pss-aileen** [first-contributions-ja/first-contributions-ja.github.io#76] [【メンテナー必読】週報]
# 次の1週間は、Lowラベルがついていないissueをさばいていく！

---

## MTGメモ目次

- [トピック: 週報は issues か Discussions にするか](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/76#discussioncomment-8843505)
  - Discussionsで
- [トピック: 最新のタスクの場所を移動させる](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/76#discussioncomment-8892308)
  - Issueは「タスク」をおく
  - issueをまとめるissueをたてて、それをpinする（（とりあえず実験的に））
  - なのでメンテナー必読issueをDiscussionsに移動（（@pss-aileenする））
  - →変更
  - 親タスク、小タスク、単独タスクのラベル、Milestoneで試行錯誤開始（20240325）
- [トピック: バージョン1リリースまでにすること、しないことを決める](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/76#discussioncomment-8848503)
  - [詳細](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/76#discussioncomment-8892381)
  - バージョン1は、現在の状態で、ロゴ、画像の用意、レスポンシブなどで「きちんと見せれる」状態にする
    - 将来的にバージョン2で変更するが、今の状態での画像もきちんと作っておく
  - バージョン2以降で、Contributor.jsonの内容を生かしたカードや、個人ページを作る
- [トピック: ロゴ、プロジェクト名](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/76#discussioncomment-8890012)
  - `First Contributions JA`
- [トピック: 次の1週間何をするか](https://github.com/first-contributions-ja/first-contributions-ja.github.io/discussions/76#discussioncomment-8848447)
  - **Priority: LowがついていないIssueをさばく！**

---

```markdown
## トピック: タイトル
- 概要

## 結論: 一言でまとめる
- 結論

## 関連 Issues、Discussions
```

- 1コメントにつき1トピック
- MTGログはリプライに残す
- Discussionsの中で強く共有したい内容があればコメントに追加しておく

---

とりあえず今回は↑のやり方でやる。
#83 にて週報の模索ログを残す

**Comment by pss-aileen** [comment]
## トピック: 週報は issues か Discussions にするか
- まだ迷っている段階
- ただ、Discussionsのほうがメモを残しやすそう

## 結論: Discussionsで！
- 引き続き、こちらでいくかんじでOK！！

**Reply by pss-aileen** [reply]
- IssueだとEditしなくちゃいけないのが面倒
- Discussionsであればメモを残しやすい

**Reply by kazzyfrog** [reply]
自分もDiscussionsの方が良いかなと思ってます！
イシューは、タスク or それに関する議論に限定したほうが、すっきりしそうです！

**Reply by pss-aileen** [reply]
- kazzyfrogさん: Discussions良い！
- pss-aileen: Discussions良い！
- kagomeさん: Pin

**Reply by pss-aileen** [reply]
追加できる

**Reply by pss-aileen** [reply]
それぞれにリプライができるかんじがよい！

**Comment by pss-aileen** [comment]
## トピック: 次の1週間何をするか
- MTGで話し合う

## 結論
- バージョン1のリリースを
  - プロジェクト名の変更
  - Contribution.jsonまで変更
  - ドックフーディンまでしてみる
  - ロゴ、関係画像も！
  - ↑ここまでできたら一旦区切る

**Reply by pss-aileen** [reply]
- バージョン1のリリースを
  - プロジェクト名の変更
  - Contribution.jsonまで変更
  - ドックフーディンまでしてみる
  - ↑ここまでできたら一旦区切る

**Comment by pss-aileen** [comment]
## トピック: バージョン1のリリースまでにすること
- 様々な導入すること、検討することが出てきた
  - #63 
  - #62 
- なので何ができれば第一弾をリリースできるようにするのか話したい
  - なんというのかな、タスクが沢山あるとどこまでをやるのかがわからなくなってくる
  - 最低限これができればリリース、あとのタスクは次のバージョンで的な感じの基準を設けたい気持ち
  - すごい個人的ですがいっぱいタスクあると焦っちゃうので...😭笑

## 結論: 
- スマホ版のリリースを含めるか、含めないか
  - **含めよう！！！！！！**
- ロゴ現在ので仮決定
  - ヘッダー
  - サムネイルOGP設定用のもの
  - リポジトリの画像（検索で出てくるやつ）
  - READMEの頭につける画像作成
- **メインタスクとサイドタスク→気にしなくてよし**
  - Priority: Highになったら頑張る！ 
- **Issueのラベル**
  - 気がついた人がつける
  Priorityは会議で話してつける

## 関連 Issues、Discussions

**Reply by pss-aileen** [reply]
- **メインタスク**
  - バージョンをリリースするのに必須のタスク
    - UIの実装
    - ドキュメントの作成・整備など 
- **サイドタスク**
  - コードの品質や内部的なこと、開発的なこと
  - バージョンにかかわらず、できる時に実装、開発していくのが良いと思っている

**Reply by kazzyfrog** [reply]
> 何ができれば第一弾をリリースできるようにするのか

これは以下のように、UXベースの最低限で良さそうですね！

- サイト上にヘッダー、ヒーローセクション、アバウトセクション、フッターがある
- サイト上から、Githubリポジトリへのリンクがある
- GithubリポジトリにREADME、CONTRIBUTING.mdがある
- 変更を加える　Contributors.jsonがある

これがあればなんとか、１連のやりたいことは出来そう？です

**Reply by kazzyfrog** [reply]
その後に、サイドタスクに着手するか、個別ページなど？の機能追加的な方向にいくかって感じですかね！

**Reply by pss-aileen** [reply]
バージョン1リリースの条件タスクが終わってきたかと思うので
今日詰めて、だいたいいつに出せるかを決められると良いかなと思う。

**Reply by pss-aileen** [reply]
- 最低限は終了？

> サイト上にヘッダー、ヒーローセクション、アバウトセクション、フッターがある
> サイト上から、Githubリポジトリへのリンクがある
> GithubリポジトリにREADME、CONTRIBUTING.mdがある
> 変更を加える　Contributors.jsonがある

できている！

**Reply by pss-aileen** [reply]
- バージョン1のリリースですること
  - この時点でリリースはするのか？
    - バージョン2とかでリリース
    - 「リリース」周知させること？→そういう認識
    - これから決めていく
- 「リリース」みんなに流していく
- 基本的な周知、リリース
  - ツイッターがメイン？
  - 記事かくか

**Reply by pss-aileen** [reply]
アクセスする媒体がPCだけなら良いが、今のままで良いが。。
スマホ勢がいるならTOPのレイアウト、モバイル対応したほうが良いよね...！！
確かに！
スマホで見れたほうがいいね、うん

**Reply by pss-aileen** [reply]
ヘッダーのロゴをGitHubからロゴへ。。？

**Reply by pss-aileen** [reply]
今あるやつで差し替える！より、よくできそうだったら更新していく方向にしてみるか？
kagomeさん,kazzyさんがロゴOK！！！うれしい！！

**Reply by pss-aileen** [reply]
ラベルを強制設定できるものがあるのか...？ないかもね...！

**Reply by pss-aileen** [reply]
Labelは気がついた人がはっていこう
Priotiyは会議のたびにはりかえたりでOKかも！

**Reply by pss-aileen** [reply]
実装の方向性がわかるものがあると良い

**Comment by pss-aileen** [comment]
## トピック: ロゴ、プロジェクト名
- プロジェクト名 first-contributions-ja で確定の方向か....！？


## 結論: First Contributions JA
- First Contributions JA
- 基本↑これで書いていく
- 堂々としてる漢字が良い！


## 関連 Issues、Discussions

**Reply by pss-aileen** [reply]
- First Contributions JA
- first-contributions-ja

**Comment by pss-aileen** [comment]
## トピック: 最新のタスクの場所を移動させる
- 

## 結論: 一言でまとめる
- メンテナー必読のIssueをDiscussionsに書き換えてみる（とりあえずやる）

## 関連 Issues、Discussions

**Reply by pss-aileen** [reply]
- https://github.com/first-contributions-ja/first-contributions-ja.github.io/issues/51 これを DiscussionsにもっていけばIssueがすこしすっきり
- IssueをまとめるIssueを作って

**Comment by pss-aileen** [comment]
## 流れ

- バージョン1として
  - Contribution.jsonまで変更
  - ドックフーディンまでして
  - ローンチじゃなくて、リリースして
  - コントリビュートができる！という状態にしたい
- バージョン2 
  - リリースブログ的なものを作る
  - ロゴも
  - 個別ページ
  - カードもTOPにいれる
  - ドキュメント類を書き換える
  - ここで、ローンチをする！（SNS等拡散）

## トピック: Contributors.json

```json
{
  "name": "Aileen",
  "githubURL": "https://github.com/pss-aileen",
  "favoriteStack": "JavaScript",
  "favoriteColor": "#333333",
  "favoriteEmoji": "🔮",
},
```

## 結論: 

> https://github.com/first-contributions-ja/first-contributions-ja.github.io/blob/main/docs/CONTRIBUTING.md

- どうしよう
- 今の方針でいくのか
- ストーリー性
- 絵文字と色のストーリー
- POP，激しい、レインボー、ごちゃごちゃ
- POPは楽しく取り組めたらいいな
- 激しい：色々なものがあっていい、おもしろい
- 自分の色
- TOPページはみんなの絵文字を降らせる
  - カード作るなら、カードごとに色、絵文字
- 個別ページ
  - 背景を好きな色、好きな絵文字ふらす
- 改めて...
  - POPな方向性
  - 色、絵文字
  - どうにかなりそう✨

上記で仮決定して、考えてみる
将来的にどう画面に表示させるかを詰めていく感じ
今のタスクをさばきつつ、画面どう表示させるか考えていく
（次1週間までに決めていく）


## 関連 Issues、Discussions

**Reply by pss-aileen** [reply]
データの削除は基本したくないので...
確定したいこと
- 内容
-

**Reply by pss-aileen** [reply]
```
{
  "name": "あなたのハンドルネーム",
  "github": "あなたのGithubアカウントのURL",
  "favoriteStack": "好きなプログラミング言語・技術スタックを１つ",
  "introduction": "自己紹介・挨拶"
},
```

**Reply by pss-aileen** [reply]
↑の内容を確定したい
これを使っての情報の見せ方を考えたい！

**Reply by pss-aileen** [reply]
- 絵文字を好きなの選んでもらって、それを上からか下からかふったり、つみあげたり
- 色！を書いてもらうのも良い！
  - 好きなのを書いてもらって、それを何かしらに適応させる
  - 例）カードの背景、ボーダーTOPで色つけたりなど

**Reply by pss-aileen** [reply]
- 好きなプログラミング言語 favoriteStack
  - これである必要もないが...
- どんなふうに表示させるかを決めたほうが全部が決まりそうなので、決めたい！

**Reply by pss-aileen** [reply]
- 絵文字、色どっちも良い！

**Reply by pss-aileen** [reply]
表示の方法にパターンをつくる...？
絵文字も個人ページ、カードに使えるようにしてもよい！
色はカードに適応させるとかでもよい！

**Reply by pss-aileen** [reply]
思ったけど微妙？
- 男性・女性、出身書いてもらう
- グラフとして出す
- ただの情報として表示するのもあり

**Reply by pss-aileen** [reply]
- グラフのアイディアよい！！！統計！！楽しい！
- 何のデータを取るか
- 独学が学校に通っているのか、エンジニアなのか？
- 見栄え（絵文字ふるのよい）

**Reply by pss-aileen** [reply]
- グラフいいなぁ
- 絵文字いいなぁ 
- 色もいいなぁ
- プロジェクト名を変える、コンセプトを決めたいといったこと
  - これがあれば、入力の基準が決まるから

**Reply by pss-aileen** [reply]
プロジェクト、派生プロジェクトとするか？
このプロジェクトで
絵文字、グラフになったりする場合雰囲気が変わる。
がらっとプロジェクトごと変えるのか、別のプロジェクトしてやる選択肢もあるのでは？

**Reply by pss-aileen** [reply]
デザインリセットして、コンセプト考えて、再度作り直し

**Reply by pss-aileen** [reply]
```
{
  "name": "Aileen",
  "githubURL": "https://github.com/pss-aileen",
  "favoriteStack": "JavaScript",
  "favoriteColor": "#333333",
  "favoriteEmoji": "🔮",
},
```

- 自己紹介いらない説？
  - なくて良いかも、好きな色とか、絵文字とかあるから
- nameがいるのか？
  - githubIdでも良いかも？
  - githubのURLの方が楽かも！
- githubのカウント

**Reply by pss-aileen** [reply]
- 絵文字をふらせる
- 色はカードとか個別のページで使う
- 好きなスタック、言語表示
- 自己紹介・挨拶

**Reply by pss-aileen** [reply]
- このかんじだと
  - スタックはいらないかも？
  - このプロジェクトに参加してくる人は初心者が多いだろう。。言語がかぎられそう
  - まだ言語を決められていないかも
  - あってもよいが、情報の質として落ちる
- 好きな技術も同じ
  - 好きな色、絵文字と同じと同じように、情報の質が落ちるような気はしない
  - かといって重要でもないけど

**Reply by pss-aileen** [reply]
- OSS活動する人は初心者が多いのか？
- みんながプログラミング初心者とは限らない？

**Reply by pss-aileen** [reply]
- macとwindowsどっち使ってる

**Reply by pss-aileen** [reply]
- 自分
  - 早く出したい
  - カード、個別はあとで良いので
  - Contriution.jsonまでで出したい
- kagomeさん
  - みんなとの兼ね合い
  - 「作りました！」で見せるときに、完成度高いほうが良いかなと思う
  - OSSとして、バージョンを重ねていくというのりがありそう、
  - 早くリリースしたい（他の人の兼ね合い）、気持ちが強い方の意見を尊重
- kazzyflogさん
  - カード、個別はなしでローンチするか
  - バージョン1ではローンチしないで、カード、個別ページでローンチする
    - ローンチ: リリースと同じ感じ？、公開しましたよ〜と周知すること 
    - リリース: 「公開」すること
  - どっちでも良い...？

**Reply by pss-aileen** [reply]
- バージョン1として
  - Contribution.jsonまで変更
  - ドックフーディンまでして
  - ローンチじゃなくて、リリースして
  - コントリビュートができる！とう状態にしたい
- バージョン2 
  - ロゴも
  - 個別ページ
  - カードもTOPにいれる
  - ドキュメント類を書き換える
  - ここで、ローンチをする！（SNS等拡散）

**Reply by pss-aileen** [reply]
- 記録をとっておきたい
  - 企業ブログにバージョン1リリースしました、前回まではこうだった、こういう変化があったよ
  - 広報？的な、ドットインストール的な変更などをお知らせしたい
  - ブログみたいなものを作りたいな！
- 派生プロジェクト、前のバージョンを残しておきつつ

**Reply by pss-aileen** [reply]
- リリースブログ
  - 書き残す方法はのち調査
  - バージョン1だしたら、そのあとはリニューアルというかお知らせとして、変更部分を出して、投稿していきたい

**Comment by pss-aileen** [comment]
Lowのラベルがついているもの以外のIssueをばりばりこなす！
